DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -71,7 +71,7 @@ class TabSwitcherViewController: UIViewController {
     var currentSelection: Int?
     
     private var tabSwitcherSettings: TabSwitcherSettings = DefaultTabSwitcherSettings()
-    private var isProcessingUpdates = false
+    private var ignoreProcessingUpdates = false
```

Comment:
I see Bunn is going to review, but my comment here is that double negatives are harder reason over.  This imo should be called `shouldProcessUpdates` then the guard in `didChange` will read more easily to understand, e.g. 

`guard shouldProcessUpdates else ... `

Reply:
I had a similar suggestion ready to go, but I prefer @brindy suggestion `shouldProcessUpdates` sounds better than what I was going to suggest `canProcessUpdates`